test/utils/evaluate-testcase.js

```diff
@@ -203,6 +203,16 @@
         expected = JSON.stringify(tokens, null, 4);
 
         try {
+            // Some variations of the options (tolerant mode, to avoid premature error)
+            esprima.tokenize(code, { tolerant: true, comment: false, loc: false, range: false });
+            esprima.tokenize(code, { tolerant: true, comment: false, loc: false, range: true });
+            esprima.tokenize(code, { tolerant: true, comment: false, loc: true,  range: false });
+            esprima.tokenize(code, { tolerant: true, comment: false, loc: false, range: true });
+            esprima.tokenize(code, { tolerant: true, comment: true,  loc: false, range: false });
+            esprima.tokenize(code, { tolerant: true, comment: true,  loc: false, range: true });
+            esprima.tokenize(code, { tolerant: true, comment: true,  loc: true,  range: false });
+            esprima.tokenize(code, { tolerant: true, comment: true,  loc: false, range: true });
```

Comment:
Now that I spot it, I realize it's not needed (the real tokenizer test uses that setting).